src/style/colors.ts

```diff
@@ -55,6 +56,15 @@ export class Colors {
     Scheme.setScheme(scheme);
   }
 
+  /**
+   * Support listening to Appearance changes
+   * and change the design tokens accordingly 
+   */
+  supportDarkMode(enable = false) {
```

Comment:
This is not clear:
1. Why is there a default for `false`?
2. If you call this (I assume in private), then I am not sure about the name, why is it `support` and not `set`?

Reply:
I changed it a little bit, no parameter needed, and I renamed it to `enableDarkMode`